### ISSUE TITLE
Unhandled exception: [Enter a title]

### ISSUE THREAD

**Issue by Mecares** [remyroy/CDDA-Game-Launcher#17]
While Patching it produces this error.

Description: [Enter what you did and what happened]
Version: 0.6.5
Type: `<class 'AttributeError'>`
Value: 'ProgressCopyTree' object has no attribute 'source_file'
Traceback:
```
  File "C:\Users\Remy\Projects\CDDA-Game-Launcher\cddagl\ui.py", line 2571, in step
  File "C:\Users\Remy\Projects\CDDA-Game-Launcher\cddagl\ui.py", line 2677, in stop

```

**Comment by remyroy** [comment]
Thanks for the report. I've fixed the bug with a23bc6aacbce9c79e1d8c5b4a2628ae50737f43f. This fix will be part of the next release which should happen soon.

**Comment by Brambor** [comment]
Same here, probably unnecessary (since, as you say, it is fixed), but here it goes:

* Description: Launched the launcher, it tried to look for latest builds. I think that this is what caused it.
* Version: 1.3.15
* OS: Windows-10-10.0.17134 (64-bit)
* Type: `<class 'TypeError'>`
* Value: '<' not supported between instances of 'NoneType' and 'str'
* Traceback:
```
  File "cddagl\ui.py", line 3169, in lb_http_finished

```
